pkg/build/api/types.go

```diff
@@ -10,8 +10,12 @@ import (
 const (
 	// BuildAnnotation is an annotation that identifies a Pod as being for a Build
 	BuildAnnotation = "openshift.io/build.name"
+	// DeprecatedBuildLabel is old value of BuildLabel, it'll be removed in OpenShift 3.1.
+	DeprecatedBuildLabel = "build"
 	// BuildLabel is the key of a Pod label whose value is the Name of a Build which is run.
-	BuildLabel = "build"
+	BuildLabel = "io.openshift.build.name"
```

Comment:
@liggitt do you think this is an OK change? I think we will make a life for our users harder as when you want to use label selector you will have to type the full namespace :-) it might look pretty ugly in CLI.

This format is a Docker images label format, but I don't we made any statement on using namespaces in `Labels` on our/k8s resources.